lib/entities/yat_record.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+
+/// Resolves YAT record
+class YatRecord {
+  String category;
+  String address;
+
+  static const yatBaseUrl = "https://a.y.at";
```

Comment:
Need to make sure that you can use testing url when its needed. For this please write `YatService` class where you could to set which url you need to use

Reply:
done